src/Modules/Data.lua

```diff
@@ -488,13 +488,28 @@ local function loadJewelFile(jewelTypeName)
 		uncompressedFileAttr.fileName = fileHandle:GetFileName()
 		uncompressedFileAttr.modified = fileHandle:GetFileModifiedTime()
 	end
+
 	fileHandle = NewFileSearch(main.userPath .. jewelTypeName .. ".zip")
 	local compressedFileAttr = { }
 	if fileHandle then
 		compressedFileAttr.fileName = fileHandle:GetFileName()
 		compressedFileAttr.modified = fileHandle:GetFileModifiedTime()
 	end
 
+	fileHandle = NewFileSearch(main.userPath .. jewelTypeName .. ".*.part*")
+	local splitFile = ""
+	if fileHandle then
+		compressedFileAttr.modified = fileHandle:GetFileModifiedTime()
```

Comment:
Doesn't this just duplicate the code above?

Reply:
I think I misread something when initially leaving this comment.  Makes sense to me now